### ISSUE TITLE
Sentence-Transformer Distillation for English to Japanese

### ISSUE THREAD

**Issue by AmitChaulwar** [UKPLab/sentence-transformers#1075]
Hi @nreimers, 

I am trying to distill a teacher model (fine-tuned on the STS dataset) to a multilingual (English and Japanese) student model. However, the translation accuracies I get are not really good (~ 50). It is the only basis for evaluation I can use as data for Japanese STS similarity data is not available. For reference, we get far better accuracy with also languages that are different than European languages (e.g. Arabic) and that also have far fewer data. For reference, we have generated our own wordpiece vocab from Wikipedia data that only contains English and Japanese words/subwords. I also pretokenised the text  using `Mecab Tokenizer` (https://github.com/Kensuke-Mitsuzawa/JapaneseTokenizers) because I observed many BERT models trained on Japanese data use it as a pretokenizer. 

Do you have any idea or experience particularly with distillation for Japanese?

**Comment by nreimers** [comment]
Hi @AmitChaulwar 
I did not do any specific experiments with Japanese. The existing multilingual model support Japanese and I think the performance is OK. I did not do any specific pre-tokenization for any language. For the XLM-R model, doing language specific pre-tokenization is not needed as it uses a sentence piece tokenizer. This could also lead to worse results as you introduce texts that the sentence piece tokenizer of XLM-R does not know.
